src/actions/ApplyAcl.ts

```diff
@@ -41,6 +43,10 @@ export async function applyServerAcls(lists: BanList[], roomIds: string[], mjoln
 
     const finalAcl = acl.safeAclContent();
 
+    if (finalAcl.deny.length !== acl.literalAclContent().deny.length) {
+        logMessage(LogLevel.WARN, "ApplyAcl", `The combined ACL has entries that would match our own server, Mjolnir will not include these but you should check your lists!`);
```

Comment:
Good idea, but let's rephrase the message a bit.

Maybe "Mjölnir has detected and removed an ACL that would exclude itself. Please check the ACL lists."